osi_common.proto

```diff
@@ -154,6 +154,9 @@ message BaseStationary
 
     /// The relative orientation of the landmark w.r.t its parent frame.
     optional Orientation3d orientation = 3;
+    
+    /// The two dimensional (flat) contour of the object. 
```

Comment:
@AndreasVIRES Thanks for the nice visualization! In our group, we define the contour in world and sensor coordinates, and not relative to the position/orientation.
@nHirsenkorn Ok, thanks for clarifying. Then I will also add the offset vectors for the rear / front axle in the vehicle description.